packages/snjs/lib/application.ts

```diff
@@ -546,13 +547,10 @@ export class SNApplication {
     this.itemManager!.setDisplayOptions(contentType, sortBy, direction, filter);
   }
 
-  public setNotesDisplayOptions(
-    tag?: SNTag,
-    sortBy?: CollectionSort,
-    direction?: SortDirection,
-    filter?: (element: SNNote) => boolean
+  public setNotesDisplayCriteria(
```

Comment:
Not sure I'm following? This function is called setNotesDisplayCriteria

Reply:
I meant that `SNApplication.setDisplayOptions()` (the more generic method that covers everything except for notes, a couple of lines above) should be renamed to `SNApplication.setDisplayCriteria()`, because while the method signature is different the intent is the same.